### ISSUE TITLE
Longer downtime kills the inverter

### ISSUE THREAD

**Issue by h0bb3** [srcfl/srcful-gateway#156]
There could be a scenario when you have been reading from the inveter and it suddenly fails (think some firmware upgrade of something) If the connection cannot be reestablished within some time there is a risk we will try to reconnect to the inverter this fails and then never reconnect again.

In this case it would make sense to actually try reconnecting for a while again... it is hard to tell when a connection should be permanently abandoned. But I would say maybe 24h is reasonable with an attempt every 15 minutes?

If this still fails a message should be issued that the connection has been permanently abandoned and that a reboot is needed to retry or the inverter would need to be reconfigured.

**Comment by h0bb3** [comment]
This would also fix #155

**Comment by h0bb3** [comment]
This is now fixed. If a inverter has been previously read we will continue trying to connect to it perpetually every 5 minutes, if another inverter is configured during this time it will override the old one and the task will die. 

This fixes eg. starting the eGW without network or loosing connection with the inverter for a longer period.

For network loss we could probably do better with some callbacks but this is a lot more complex.

A caveat could be that a new Inverter should be created if the open fails - we have experienced some issues with reconnection to some real inverters. But we'll do that if we actually get this problem.
